Makefile

```diff
@@ -139,13 +139,15 @@ check-clippy:	install-clippy
 	cd src/client; cargo +$(RUSTV) clippy --all-targets  -- -D warnings
 
 
-build-all-test:	
+build-all-test:
 	cargo build --tests --all-features
 
 run-all-unit-test:
 	cargo test --all-features
 
 install_musl:
+	sudo apt-get update;
```

Comment:
this need to be specifically target toward ubuntu which make it not suitable here